tests/components/abode/test_alarm_control_panel.py

```diff
@@ -16,6 +16,7 @@
     STATE_ALARM_ARMED_HOME,
     STATE_ALARM_DISARMED,
 )
+from homeassistant.helpers import entity_registry as er
```

Comment:
I'm personally not a fan of aliasing imports unless it is to avoid collisions.

```suggestion
from homeassistant.helpers import entity_registry as er
```

I think seeing `er` and `dr` around the code, is far less clear compared to `entity_registry` or `device_registry`.

Reply:
Nevermind, I can see why. There is actually a name collision in most cases 👍

Reply:
The problem is that `area_registry`, `device_registry` and `entity_registry` all have helpers with same or similar names, in particular `*_registry.async_get()`, so we need to import the module to be clear on which function is called without having to alias each imported function.
Then we will need to do the equivalent to: `entity_registry = entity_registry.async_get()` everywhere, with an obvious collision in the naming. 

This is one way to avoid those collisions, others are creative naming of the variables `ent_registry = entity_registry.async_get()`, `entity_reg = entity_registry.async_get()` and so on.

Not sure which is the better option.

Reply:
Yeah I came to the same conclusion along with the reviews 👍